### ISSUE TITLE
marking done modifies freetime for that day

### ISSUE THREAD

**Issue by jeremyjs** [chetaldrich/Dunmo#22]
if you mark a task done, it subtracts the remaining time from your freetime
if you then mark it undone, you will not regain the time in the day
probably, marking it "done" should not modify your freetime unless you are doing a pomodoro

**Comment by lbiester** [comment]
I'm not sure that this should be true. I think if you finish something, it should definitely subtract the amount of time spent on it (presumably the amount of time you said you needed) from the amount of time you have left in the day.

**Comment by jeremyjs** [comment]
Right, so the way it currently is, whenever you click save from the pomodoro screen, it subtracts the amount of time you just then spent. If you are back on the task screen, clicking done should not subtract additional time. If the user marks a task as done from the tasks screen, I'm saying we should assume they haven't spent any more time. Basically, if they want to track time, they should use the timer.
